costmap:
  width: 40
  height: 40
  resolution: 0.1
  decay_factor: 0.1

inflation_radius: 0.3

base_frame: "base_link"
scan_frame: "odin1_base_link"

dwa:
  robot_radius: 0.2
  obstacle_margin: 0.1

  allow_backward: false
  enable_rotate_recovery: true

  heading_align_thresh: 0.5
  heading_boost: 2.0

  max_vel_x: 1.3
  max_omega: 1.3
  max_acc_x: 10.0
  max_alpha: 10.0

  sim_time: 2.0
  sim_dt: 0.1
  v_samples: 6
  w_samples: 11

  no_simulation: true

  weights:
    clearance: 3.0
    path: 2.0
    heading: 1.0
    velocity: 0.5
 